shell/app/layout/pages/page-container/components/sub-sidebar.scss

```diff
@@ -35,3 +35,102 @@
     content: none;
   }
 }
+
+.side-nav-menu {
+  transition: width 0.3s cubic-bezier(0.2, 0, 0, 1) 0s;
+  scrollbar-width: 0;
+  .menu-container {
+    &::-webkit-scrollbar {
+      width: 6px;
+      height: 6px;
+      display: none;
+    }
+    &::-webkit-scrollbar-track-piece {
+      display: none;
+    }
+    &:hover {
+      &::-webkit-scrollbar {
+        display: block;
+      }
+    }
+    &::-webkit-scrollbar-thumb {
+      border-radius: 8px;
+      opacity: 0.6;
+      background: $color-dark-4;
+    }
+
+    &::-webkit-scrollbar-track {
+      border-radius: 8px;
+      background: transparent;
+    }
+  }
+
+  .ant-menu {
+    background-color: $white;
+    .ant-menu-submenu-arrow {
+      color: rgba(9, 24, 33, 0.44);
+      font-weight: 500;
+    }
+    .ant-menu-item-icon {
+      color: $primary;
+      font-size: 16px;
+    }
+
+    .ant-menu-item {
+      margin: 0;
+      &.ant-menu-item-selected {
+        background-color: $color-tree-bg-active;
```

Comment:
This variable not exist in core, should move files such as `_color.scss` to core.